components/external-link/style.scss

```diff
@@ -5,4 +5,13 @@
 		margin: 0 .2em;
 		vertical-align: top;
 	}
-}
\ No newline at end of file
+
+	&.components-icon-button {
+		display: flex;
+		align-items: center;
+	}
+
+	.wp-core-ui .editor-block-list__layout & {
```

Comment:
Components should be generic and not assume anything of their surrounding context. If styles are needed, it should be applied as a descendent selector from `block-list/style.scss`.

Reply:
> build-in logic of the ExternalLink when passed both a href and target="_blank" prop.

OK. Worth noting the target value can also be a named target, as Gutenberg does for the Preview button e.g. `target="wp-preview-7169"`